ground/gcs/src/plugins/telemetryscheduler/telemetryschedulergadgetwidget.cpp

```diff
@@ -1089,7 +1089,6 @@ void QFrozenTableViewWithCopyPaste::init()
                                    "color: #FF0000;"
                                    "background-color: #8EDE21;"
                                    "selection-background-color: #999}"); //for demo purposes
-    frozenTableView->setSelectionModel(selectionModel());

```

Comment:
PS. It appears to work fine. ;-)
<img width="760" alt="untitled" src="https://cloud.githubusercontent.com/assets/9995998/10858807/a249aed8-7fbc-11e5-8005-0bd5f31a3050.png">